pkgs/applications/misc/navit/default.nix

```diff
@@ -21,34 +31,54 @@ stdenv.mkDerivation rec {
     sha256 = "0vg6b6rhsa2cxqj4rbhfhhfss71syhnfa6f1jg2i2d7l88dm5x7d";
   };
 
-  #hardeningDisable = [ "format" ];
-  NIX_CFLAGS_COMPILE = [ "-I${SDL.dev}/include/SDL" ];
+  patches = [ ./CMakeLists.txt.patch ];
+  hardeningDisable = [ "format" ];
```

Comment:
Why is this necessary again?

Reply:
i don't understand well and i restore from the original navit since it was not failing.

Reply:
But it was commented out previously.